### ISSUE TITLE
v1 todo

### ISSUE THREAD

**Issue by PaulLeCam** [PaulLeCam/react-leaflet#87]
## Must have

- [x] UMD build ([#29](https://github.com/PaulLeCam/react-leaflet/issues/29))
- [x] Support for Controls ([#81](https://github.com/PaulLeCam/react-leaflet/issues/81))
  - [x] Zoom
  - [x] Attribution
  - [x] Layers
  - [x] Scale
- [x] Contributing guidelines
  - [x] Setup jsbin/jsfiddle with dependencies to reproduce bugs (http://jsfiddle.net/paul_lecam/q2v7t59h/)
  - [x] Add `CONTRIBUTING.md`
  - [x] Add `ISSUE_TEMPLATE.md`
- [x] Remove support for `onLeaflet...` events
  - [x] Add deprecation warning in previous release
- [x] Extract docs out of readme, split in different sections
- [x] Remove support for `getLeafletElement()` method
  - [x] Add deprecation warning in previous release
- [x] Support only React v15+
- [ ] Leaflet v1 support ([#66](https://github.com/PaulLeCam/react-leaflet/issues/66))

## Nice to have

- [ ] Website
  - [ ] Getting started
  - [ ] API
  - [ ] Examples
- [ ] Examples for each component
- [ ] Advanced examples (Redux, Rx...)
- [ ] Tutorial for creating custom components
  - [ ] Reference implementation for plugin integration

**Comment by jonboiser** [comment]
Hi Paul,

I like the potential of this library for quickly bootstrapping geospatial apps written in React. Are there are any tasks you can use help with?

**Comment by PaulLeCam** [comment]
Hi @jonboiser, thanks for asking!

I think adding the attribution, layers and scale controls is the next priority. It shouldn't be a lot of work, just make sure they behave as expected.

Leaflet v1 support is tricky because I want to have it as soon as it's stable, but the current betas seem to change the APIs so I'm not sure it's worth investing time on it.

**Comment by jonboiser** [comment]
Cool. I'll take a look at how you've implemented the Zoom control. I think I can help with the docs and website, once I play around with the library. At my company, we started a similar project for the ArcGIS JS API and Leaflet, but think we should deprecate the Leaflet piece, since this library is available.

**Comment by jonboiser** [comment]
I pushed some code for the AttributionControl to my fork. It seems like coding the other controls will be pretty much the same. I'll tackle the others during the week and submit a PR.

**Comment by snario** [comment]
What's the progress like on this?

**Comment by PaulLeCam** [comment]
Mostly waiting on a RC of Leaflet v1 to start adapting what needs to be, and the final v1 release to effectively version this lib as v1 as well.

**Comment by jampy** [comment]
Any news on this, since meanwhile there is even a [second RC](http://leafletjs.com/2016/07/16/leaflet-1.0-rc2.html) available?

**Comment by PaulLeCam** [comment]
Closing this issue, please go to the [v1 release plan](https://github.com/PaulLeCam/react-leaflet/issues/197) for more information about this future release.
